scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -128,10 +129,10 @@ package object sparkey {
      *
      * @return A singleton SCollection containing the [[SparkeyUri]] of the saved files.
      */
-    def asSparkey: SCollection[SparkeyUri] = {
+    def asSparkey(implicit w: SparkeyWritable[K, V]): SCollection[SparkeyUri] = {
       val uuid = UUID.randomUUID()
       val basePath = self.context.options.getTempLocation + s"/sparkey-$uuid"
-      this.asSparkey(basePath)
+      this.asSparkey(basePath)(w)
```

Comment:
I'm surprised that this is needed. I thought `implicit` params should propagate automatically? Same with `def asSparkeySideInput` down there.